flask_resty/pagination.py

```diff
@@ -481,10 +481,13 @@ def decode_cursor(self, cursor: str) -> Tuple[str, ...]:
     def decode_value(self, value: str):
         value = value.encode("ascii")
         value += (3 - ((len(value) + 3) % 4)) * b"="  # Add back padding.
-        value = base64.urlsafe_b64decode(value)
-        return value.decode()
+        value = base64.urlsafe_b64decode(value).decode()
+
+        return None if value == "None" else value
```

Comment:
it's a bit odd to string-match against `None`; it might make sense to encode `None` as `null` or `NULL` in `encode_value`. wdyt?

Reply:
I like that

Reply:
I think the idea would be to decode it to `None` or `sql.null()` still. But serialize to the as "null" for the cursor.

I think we don't need to do it in this PR tho

Reply:
I prefer to work with python data types whenever possible, mixing with sql.null can be confusing, I never know if I should use one or another. But  it's SQLA thing.
BTW can we change to something like
```return value if value is None else base64.urlsafe_b64decode(value).decode()```
so we don't need to compare to `"None"` (it looks odd)? Or is there a some hidden middle step that prevent this?

Reply:
some value needs to be encoded in the cursor or it won't work correct. `null` is the actual value here, `None` is simply the python representation of it, hence it makes sense to serialize the None as `'null'` in the same way None serialized to the DB `null` in a query

Reply:
which is to say this decode_value function is decoding sql column values that were stored in a b64'd string